pages/security/message-security/openpgp/gpg-best-practices/en.text

```diff
@@ -354,6 +354,14 @@ gpg> save
 
 h2. Additional suggestions.
 
+h3. Consider using an ECC subkey
+
+Since gnupg 2.1 ECC subkeys are supported.
+
+They are faster, which is very useful for security tokens. Note that if there's a RSA and ECC key on a key, GnuPG 1.4 will do the right thing and choose the one that works (e.g. use RSA if it doesn't support ECC).
```

Comment:
"key on a key" is a bit hard to parse. Maybe this could be re-worded to "Note, if there both a RSA and ECC key available, GnuPG ...."

Reply:
Also, you say GnuPG 1.4 does the right thing, this isn't supposed to be 2.1 (i'm unsure, I just see the next section pointing out more details for 2.1)